### DISCUSSION TITLE
Battery 100% & constantly switching small discharge without need (BLE-Branch)

### DISCUSSION THREAD

**Discussion by HotteX** [Louisvdw/dbus-serialbattery#548] [Q&A] [answered]
Hi, 

I think this was discussed before, but I cannot find it again. 

I run currently with the -ble branch and the battery never "rests" when full. (Same with Linear on or off with my short test)

SOC 100%.
DC Voltage is jumping a little and DC Current hops between 0.0A and -0.6A constantly. 
The system should be at rest I think as my Solar Inverter has a constant surplus of > 1.000 Watts.
Multiplus does it now since about 3 hours, so I guess it won't stop.

**Comment by mr-manuel** [comment, marked as answer]
This is because the system stability is counting first and the battery is used as buffer. The current will never be 0 except you diconnect it. This is the behaviour of the MultiPlus and cannot be changed.

**Reply by HotteX** [reply]
Ok. 
I had the 0.14.3 running before with steps enabled (but all values set in a way they don't interfere, basically pushing 29A until the maximum cell voltage was reached (set at a safe 3.4V...cells never went beyond 3.5V) and I am pretty sure the battery did not show a current when full. Is there a difference and is it "logical" or should I recheck my old settings to find out why there was no discharge (or why I didn't see it, while it was there).

What I had set was
CCCM_SOC_ENABLE = False
CVCM_ENABLE = False
because it seemed interfere with my non-balanced system (in testing) a lot.
Might make no sense...I was in a hurry and still testing. 

Why I am asking is: I was hoping to get the Multiplus into Search mode to conserve power when it's not needed (because battery is full and solar power AC connected)...it sometimes sits for 10 hours without anything to do. 
Might be a wrong thought... I am quite new in the battery business.

Thanks a lot for caring so much for the project while Louis is away!

**Reply by mr-manuel** [reply]
This is a good question with the search mode. I think the best would be, if you ask this in the Victron Community.
